packages/vendor/package.json

```diff
@@ -39,26 +39,26 @@
     "postpublish": "npm run deploy:docs"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@babel/preset-env": "^7.7.0",
+    "@babel/core": "^7.15.8",
+    "@babel/preset-env": "^7.15.8",
     "@theforeman/vendor-core": "^8.16.0",
-    "babel-loader": "^8.0.6",
-    "compression-webpack-plugin": "^2.0.0",
+    "babel-loader": "^8.2.3",
+    "compression-webpack-plugin": "^6.1.1",
```

Comment:
foreman minimum: `NodeJS 10 or newer`  - https://www.theforeman.org/manuals/3.0/index.html#3.InstallingForeman 
`minimum supported Node.js version is 10.13` - https://github.com/webpack-contrib/compression-webpack-plugin/blob/master/CHANGELOG.md#400-2020-05-12
Should we update the foreman documentation?